_spinnaker/terraform_integration.md

```diff
@@ -13,133 +13,30 @@ If you decide to enable this feauture and have any feedback you'd like to submit
 
 ## Prerequisites
 
-1. A Armory Spinnaker installation running on Kubernetes and installed via Armory Halyard. ([instructions](/spinnaker/install)).
+1. A Armory Spinnaker installation running on Kubernetes and installed via Armory Halyard. ([instructions](/spinnaker/install)). If you haven't updated Armory Halyard in a while, you'll need to do so to get access to these new features.
 2. A Github API token that has access to your Terraform project. See [this documentation](https://blog.github.com/2013-05-16-personal-api-tokens/) for details on how to generate this token.
-3. An AWS keypair with with enough permission to access your state store in S3 and manage the resources created by Terraform.
+3. Pre-release candidate of Armory Spinnaker. At the time of this writing, `2.1.1-rc2582` contains the first alpha release of our Terraform integration. You can view a list of all current release candidates using `hal version list --release rc`.
 
 ## Installation
 
-#### Deploying the Terraformer service
-
-The core of our Terraform integration is the Terraformer service. This is the service which is responsible for fetching your Terraform code and executing it. To do this, we'll need to install the following Kubernetes manifest using `kubectl`. *Note: we'll be adding support for deploying Terraformer via Halyard in future releases.* 
-
-You'll need to replace 4 references before doing so:
-1. `your-github-api-token` - the API token which will be used to fetch your repository
-2. `profile-name` - the name of the AWS profile which your Terraform code will use to provision infrastructure
-3. `your-aws-access-key` - the AWS Access Key ID of your keypair
-4. `your-aws-secret-access=key` - the AWS Secret Access Key of your keypair
+### Enabling the Terraform integration
 
+You can enable the Terraform integration via Armory Halyard. Before you do, be sure you have a Github access token. This will enable the Terraform integration to interact with entire Github repositories.
```

Comment:
Does terraformer work with Bitbucket? Reltio only uses bitbucket.

Reply:
I've only tried it with Github but it shouldn't be difficult to make it work with Bitbucket. Might require a little code change.